UI/css/stylesheet.css

```diff
@@ -1049,6 +1060,23 @@ header .navbar {
 /******** *****************************
 **************END DASHBOARD ***********/
 
+/* UPDATE SIGNUP AND SIGNIN */
+#login-form span, #signup-form span, #reset-form span{
+    position: absolute;
+    margin-top: -1rem;
+}
+.border-radius{
+    border-radius: 10px;
+}
+#login-form button{
+    margin-top: 1.1rem;
+    background: #000;
+}
+
+/* NEW FEATURE, RESET PASSWORD */
```

Comment:
Refactor this file, Create different CSS files for all Html files that perform different functionalities.
For example the index page and the login page should have different CSS files.

Reply:
Thank you i am going to do it right now.